plugins/fs/src/commands.rs

```diff
@@ -3,6 +3,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // SPDX-License-Identifier: MIT
 
+use anyhow::Ok;
```

Comment:
```suggestion
```

Do we really need this? If so, which line does rust complain about if it's not there?

Reply:
Haha, I didn't even notice using it